src/stories/standalone/SignaturePad/SignaturePad.story.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import "../../../i18n";
+import SignaturePad from "../../../standalone/SignaturePad/index";
+import { select, text, boolean, withKnobs } from "@storybook/addon-knobs";
+import { action, withActions } from "@storybook/addon-actions";
+
+export const SignaturePadStory = (): React.ReactElement => {
+	const penColor = ["blue", "black", "red", "yellow", "pink"];
+	const disabled = boolean("disabled", false);
+	const setSignature = text("setSignature", "");
+	const getSignature = action("getSignature");
+
+	return (
+		<SignaturePad
+			penColor={select("Pen Color", penColor, "blue")}
```

Comment:
I assume this supports HEX colors, there is a color knob for this.

Reply:
yes, it supports HEX colors